add-on/src/lib/ipfs-companion.js

```diff
@@ -15,6 +16,12 @@ const createRuntimeChecks = require('./runtime-checks')
 const { createContextMenus, findUrlForContext } = require('./context-menus')
 const createIpfsProxy = require('./ipfs-proxy')
 
+// LOG
+// production: localStorage.debug = 'ipfs-companion:*,-*:debug'
+//        dev: localStorage.debug = 'ipfs-companion:*'
+localStorage.debug = 'ipfs-companion:*,-*:debug'
```

Comment:
Could this come from the manifest? Or some sort of user config/env var?

Reply:
Btw, in my libdweb branch I had to do this to enable logging: https://github.com/ipfs-shipyard/ipfs-companion/pull/553/files#diff-fc43b78eb6e482a2ad1a6924959d9ec1R4

Does this definitely work in Chrome?

Reply:
🙄 I'm really bad at reading the PR description...

Reply:
> Could this come from the manifest? Or some sort of user config/env var? 

I hoped  we could override it via `web-ext run`'s  [`--pref`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/web-ext_command_reference#--pref) (or via env: `$WEB_EXT_PREF`) but this parameter works only for browser's settings from `about:config`.
WebExtension's _Preferences_ are now kept in separate storage thet can't be overridden on the fly without additional code.

As a workaround,  I was thinking about _Preferences_ screen in extension itself: adding a simple checkbox (that adds or removes `-*:debug`) or  an input which enables you to edit `localStorage.debug` (and enable logs from js-ipfs, libp2p etc).   Personally I'd go with input, as it gives more control to developers and power users who want to look under the hood.   As long we keep it in _Experiments_ section it should be fine. 
@alanshaw  what do you think? does it feel useful?